File: sample/src/main/java/com/ms_square/debugoverlay/sample/IPAddressModule.java
package com.ms_square.debugoverlay.sample;

import android.content.Context;

import androidx.annotation.NonNull;

import com.ms_square.debugoverlay.OverlayModule;
import com.ms_square.debugoverlay.modules.SimpleViewModule;

public class IPAddressModule extends OverlayModule<String> {

    public IPAddressModule(@NonNull Context context) {
        super(new IPAddressDataModule(context), new SimpleViewModule(R.layout.view_overlay_ip));
    }
}

File: sample/src/main/kotlin/com/ms/square/debugoverlay/sample/IPAddressModule.kt
package com.ms.square.debugoverlay.sample

import android.content.Context
import com.ms.square.debugoverlay.OverlayModule
import com.ms.square.debugoverlay.modules.SimpleViewModule

class IPAddressModule(context: Context) :
  OverlayModule<String>(
    IPAddressDataModule(context),
    SimpleViewModule(R.layout.view_overlay_ip)
  )
